configs/aws-service-operator.yaml

```diff
@@ -234,4 +234,5 @@ items:
             - server
             - --cluster-name=<CLUSTER_NAME>
             - --region=<REGION>
-            - --account-id=<ACCOUNT_ID>
\ No newline at end of file
+            - --account-id=<ACCOUNT_ID>
+            - --k8s-namespace=<K8S_NAMESPACE>
```

Comment:
This is also a code generated file so you will need to modify https://github.com/awslabs/aws-service-operator/blob/master/code-generation/pkg/codegen/assets/aws-service-operator.yaml.templ#L112